pkg/certmanager/certmanager.go

```diff
@@ -50,10 +50,18 @@ type Options struct {
 	IssuerRef cmmeta.ObjectReference
 }
 
-// Manager is used for signing CSRs via cert-manager. Manager will create
+type Interface interface {
```

Comment:
Suggestion: call this `Signer` rather than `Interface`

Reply:
Thanks @irbekrm. 

This is actually an intentional change. I've come around to the idea that `signer.Signer` is bad because it is stuttering, whereas `signer.Interface` makes sense and doesn't stutter.

Reply:
I guess `certmanager.Signer` would not be a stutter and may be expressing the intent more clearly, but `Interface` is ok too imo

Reply:
This is true, I will change it to ‘Signer’.

Reply:
Done in https://github.com/cert-manager/istio-csr/pull/75/commits/8550a4c187ff733d02423553413fdbefba4cdce8